parsers/cxx/cxx_parser.c

```diff
@@ -1505,12 +1515,14 @@ void cxxParserAnalyzeOtherStatement(void)
 			// out its proper scope. Better avoid emitting this one.
 			CXX_DEBUG_PRINT("But it has been preceded by the 'friend' keyword: this is not a real prototype");
 		} else {
-			int piCorkQueueIndex;
-			int iScopesPushed = cxxParserEmitFunctionTags(&oInfo,CXXTagKindPROTOTYPE,CXXEmitFunctionTagsPushScopes, &piCorkQueueIndex);
-			if (piCorkQueueIndex != CORK_NIL)
+			int iCorkQueueIndex, iCorkQueueIndexFQ;
```

Comment:
I'm not sure whether @pragmaware cares or not but I have a comment about this change.

This change, renaming piCorkQueueIndex to iCorkQueueIndex, is correct but this change is not needed for filling end fields of FQ tags.
So before introducing changes for filling end fields of FQ tags, I would like you to make a small separated commit only for fixing the wrong variable name, and add the commit as "preparation commit" for this pull request.